plugins/jobs/src/js/components/form/helpers/JobFormData.ts

```diff
@@ -73,7 +91,10 @@ export interface JobRun {
   user?: string;
   taskKillGracePeriodSeconds?: number;
   restart?: JobRestart;
+  activeDeadlineSeconds?: number;
   volumes?: Array<JobVolume | SecretVolume>;
+  restartJob?: boolean;
+  retryTime?: number;
```

Comment:
I think `restartJob` and `retryTime` should be removed from the interface. They are contained in `restart` and shouldn't be root level properties in this object.

Reply:
have been removed! thanks for the find!